docs/src/components/HeadingWithLink.tsx

```diff
@@ -1,24 +1,43 @@
 import React from "react";
-import styled from "styled-components";
-import { ButtonLink, Stack } from "@kiwicom/orbit-components";
+import styled, { css } from "styled-components";
+import { Stack } from "@kiwicom/orbit-components";
 import { Link as LinkIcon } from "@kiwicom/orbit-components/icons";
 import { SpaceAfter } from "@kiwicom/orbit-components/lib/common/common";
 
 import { getTextFromChildren, slugify } from "../utils/common";
 
 export const StyledAnchorWrapper = styled.div`
-  margin-top: -10px;
-
   svg {
     opacity: 0;
-    transition: opacity ${({ theme }) => theme.orbit.durationNormal} ease-in;
   }
 
   &:hover svg {
     opacity: 1;
   }
```

Comment:
This can be moved in `StyledAnchor`, so styles there would look something like:

```
svg {
  opacity: 0;
}

${StyledAnchorWrapper} {
  &:hover, &:focus {
    svg {
      opacity: 1;
    }
  }
}
```